pkg/cmd/issue/create/create_test.go

```diff
@@ -944,6 +1025,15 @@ func TestIssueCreate_projectsV2(t *testing.T) {
 	defer http.Verify(t)
 
 	http.StubRepoInfoResponse("OWNER", "REPO", "main")
+	http.Register(
+		httpmock.GraphQL(`query Repository_fields\b`),
+		httpmock.StringResponse(`{ "data": { "Repository": { "fields": [
+			{ "name": "autoMergeAllowed" },
+			{ "name": "projects" },
+			{ "name": "pullRequestTemplates" },
+			{ "name": "visibility" }
+		] } } }`),
+	)
```

Comment:
Since this is repeated so much, do you think it makes sense to extract it to a named helper function? That might also help give more context on what it's doing, too